tests_spec/eibc.md

```diff
@@ -2,21 +2,21 @@
-| 5  | Rollapp token ibc-transfer timeout | Same as the case above | 🛑 <br> (missing) | <ul><li> send rollapp IBC denom from dymension to rollapp </li> <li> Same as the case above </li></ul> | Partly solved <br> (lack query `packet commitment` left on the rollapp) |  [ibc_timeout_test](../tests/ibc_timeout_test.go)  |
+| 1  | Rollapp token Demand order is created upon memo submission and fulfilled |At least 2 rollapps running, Rollapp A and B. Rollapp B is at finalized height < Rollapp A finalized height. Rollapp B has a channel-id different from the hub-channel-id. | 🛑 <br> (missing) | Demand order for Rollapp tokens created and fulfilled upon memo submission. | Partly solved <br> (lack query `packet commitment` left on the rollapp) | [TestEIBCFulfillment_EVM](../tests/ibc_grace_period_test.go#28) [TestEIBCFulfillment_Wasm](../tests/eibc_fulfillment_test.go#322) |
+| 2  | DYM Demand order is created upon memo submission and fulfilled | Same as the case above | 🛑 <br> (missing) | Timeout triggered for EIBC transaction from hub to rollapp due to non-finalization. | Partly solved <br> (lack query `packet commitment` left on the rollapp) | [TestEIBCTimeoutHubToRollapp](../tests/eibc_timeout_test.go#24) |
+| 3  | 3rd party token (i.e blumbus) Demand order is created upon memo submission and fulfilled | Same as the case above | 🛑 <br> (missing) | <ul><li> Eibc demand order created</li><li> Fulfill the eibc demand order</li><li> Verify the originator gets the funds minus the fee right away</li><li> Verify the fulfiller gets the full amount after the grace period is finalized</li><li> No packet commitment left on the rollapp (i.e. ack received). Can be queried using the CLI rollapp-evm q ibc channel packet-commitments transfer \<channel-id></li></ul>| 🛑 <br> (missing) | [TestEIBCFulfillment_ThirdParty_EVM](../tests/eibc_fulfillment_test.go#1006) [TestEIBCFulfillment_ThirdParty_Wasm](../tests/eibc_fulfillment_test.go#1405)|
+| 4  | DYM ibc transfer timeout | Same as the case above | 🛑 <br> (missing) | <ul><li> send **dym** from dymension to rollapp </li> <li>set minimum packet timeout (i.e. 1ms) so that the packet is not relayed before it is picked by a relayer  </li><li> sender should get funds back after grace period. </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id></li> <li> demand order is created with correct ibc denom and fee </li> <li>demand order can get fulfilled </li> <li>upon timeout the funds should go to the fulfiller </li> </ul>| Partly solved <br> (lack query `packet commitment` left on the rollapp) | TODO |
```

Comment:
Why we make this one TODO, I thought we've already do it on timeout test?

Reply:
Nice catch. Didn't see we cover 2 cases there, updated